news/2018-06-02-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,345 @@
+---
```

Comment:
Some links are using old site and some are using new site, would be better to have consistency